normandy/selfrepair/static/js/self_repair_runner.js

```diff
@@ -45,6 +45,36 @@ function loadAction(recipe) {
 }
 
 /**
+ * @promise {Object} The data to send to fetch_bundle to identify this client.
+ */
+function get_fetch_recipe_payload() {
```

Comment:
Doh. `camelCase`, most likely. I clearly didn't have all my screws in when I wrote this code.